subprojects/docs/src/samples/userguideOutput/taskFinalizersWithFailure.out

```diff
@@ -1,2 +1,18 @@
 taskX
 taskY
+
+FAILURE: Build failed with an exception.
+
+* Where:
+Build file '/home/user/gradle/samples/userguide/tasks/finalizersWithFailure/build.gradle' line: 4
+
+* What went wrong:
+Execution failed for task ':taskX'.
+> java.lang.RuntimeException (no error message)
+
+* Try:
+Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
+
+* Get more help at https://help.gradle.org
+
+BUILD FAILED in 0s
```

Comment:
It doesn't seem like any transformation is done to prevent slow down and not been able to match `BUILD FAILED in 1s` for example. From the output matcher (`AnyOrderOutputMatcher`/`SequentialOutputMatcher`) and what was previously there, I think this line should be left out so it doesn't cause flakiness. The same goes for the other output samples.

Reply:
The output is been normalized here: https://github.com/gradle/gradle/blob/5fbec26ae49a53a28256e56086faa4a2e80c09f3/subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/OutputScrapingExecutionResult.java#L147